boards/arm/nrf52_pca10040/nrf52_pca10040.dts

```diff
@@ -121,23 +148,20 @@ arduino_i2c: &i2c0 {
 &spi0 {
 	/* Cannot be used together with i2c0. */
 	/* status = "okay"; */
-	sck-pin = <27>;
-	mosi-pin = <26>;
-	miso-pin = <25>;
 };
 
 arduino_spi: &spi1 {
 	status = "okay";
-	sck-pin = <31>;
-	mosi-pin = <30>;
-	miso-pin = <29>;
```

Comment:
Instead of replacing these assignments (which are fine, `spi1` and `i2c1` cannot be used together, so it seems reasonable to assign pins 30 and 31 as default ones for both of them), it will be better to move the `arduino_spi` label to the `spi2` node and correct only the pin assignments there.